src/Kunstmaan/AdminBundle/Helper/Security/OAuth/OAuthUserCreator.php

```diff
@@ -56,7 +58,11 @@ public function getOrCreateUser($email, $googleId)
             }
 
             foreach ($this->getAccessLevels($email) as $accessLevel) {
-                $user->addRole($accessLevel);
+                /** @var GroupInterface $group */
+                $group = $this->em->getRepository('KunstmaanAdminBundle:Group')->findOneBy(['name' => $accessLevel]);
+                if (null !== $group) {
```

Comment:
Maybe we need some fallback so the process doesn't create a user without a role or group and effectively makes the username unusable. Some projects might have custom groups, whereas you can always be sure the roles will be the same.

Reply:
Hmm, you configure the groups you want to add in the config.yml so it's your own responsibility to check if the group has roles in it IMHO.